typing/typeclass.ml

```diff
@@ -1685,32 +1684,32 @@ let class_infos define_class kind (* TODO: tuple to record *)
     )
 
 let extract_type_decls {
-      clsres_clty = clty;
-      clsres_cltydef = cltydef;
-      clsres_obj_id = obj_id;
-      clsres_obj_abbr = obj_abbr;
-      clsres_cl_abbr = cl_abbr;
-      clsres_req = req
+      clty;
+      cltydef;
+      obj_id;
+      obj_abbr;
+      cl_abbr;
+      req
      } decls =
   (obj_id, obj_abbr, cl_abbr, clty, cltydef, req) :: decls
 
 let merge_type_decls cls_res (obj_abbr, cl_abbr, clty, cltydef) =
```

Comment:
Would `decls` be a better naming than `cls_res`?

Reply:
The name of the function I think is already a bit strange (not yours, I realise) - it has a plural name, but it's actual a singular operation used with `map2` further down. Perhaps rename it `decl`?